### PR TITLE
Updating the index.md with arcconf information

### PR DESCRIPTION
**Pull request by brraghav**

### CODE REVIEW

**Review by cathay4t — CHANGES_REQUESTED** [review]
@brraghav I am afraid you get the markdown syntax wrong.
You should use something like:
```markdown
* [Microsemi Adaptec and Smart RAID][29]

[29]: doc/array_conf/Arcconf_Conf.html
```

Please at least try to run the github-pages locally to make sure your changes is correct.

**Comment by brraghav** [comment]
@cathay4t Thanks for the syntax help with the markdown. I understood it now. I debugged with the jekyll and have added the changes to the same PR.

**Comment by brraghav** [comment]
@cathay4t  Microsemi bundles the arcconf with StorMan package. Hence the installation for StorMan is included in the webpage for the arcconf installation. I cross verified this with User guide of arcconf.

**Comment by brraghav** [comment]
@cathay4t  I am still making some changes. I will ping you after that for review..

**Comment by brraghav** [comment]
@cathay4t Please review this PR. I debugged this using the jekyll.
Kindly let me know if any changes required.

File: doc/array_conf/Arcconf_Conf.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Arcconf SmartArray Configuration Guide
```
Comment (cathay4t) [inline doc/array_conf/Arcconf_Conf.md]: Should be `Microsemi SAS Controllers Configuration Guide`

File: doc/array_conf/Arcconf_Conf.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+---
+LibStorageMgmt can manage Arcconf SmartArray via arcconf plugin
+(`libstoragemgmt-arcconf-plugin` package).
+You are also required to install `arcconf` binary tool from Microsemi
+website.
```
Comment (cathay4t) [inline doc/array_conf/Arcconf_Conf.md]: The `arcconf` tool is also included in `Microsemi maxView Storage Manager` tool.

File: doc/array_conf/Arcconf_Conf.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+* Execute this commands to install arcconf executables:
+
+```
+./StorMan-2.05-22885.x86_64.bin
+```
+
+Above steps were tested on:
+
+* `RHEL 7.1 x86_64`
```
Comment (cathay4t) [inline doc/array_conf/Arcconf_Conf.md]: Please try it on RHEL 7.4 or at least RHEL 7.3.

**Review by cathay4t — CHANGES_REQUESTED** [review]
Please let me find a server and actually try it in lab.

File: doc/array_conf/Arcconf_Conf.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+* Execute this commands to install arcconf executables:
+
+```
+./StorMan-2.05-22885.x86_64.bin
+```
+
+Above steps were tested on:
+
+* `RHEL 7.3 x86_64`
+* `StorMan-2.05-22885.x86_64.bin`
```
Comment (cathay4t) [inline doc/array_conf/Arcconf_Conf.md:43]: I failed to find 2.05, but only found `msm_linux_x64_v2_04_22665.tgz` on the website.

And we should suggest user to use rpm instead:

```bash
tar xf msm_linux_x64_v2_04_22665.tgz
sudo yum install ./cmdline/rpm/Arcconf-2.04-22665.x86_64.rpm
```

**Review by cathay4t — APPROVED** [review]
Besides trivial lints, I approve this PR.

**Comment by cathay4t** [comment]
@brraghav Could you update the patch regarding https://github.com/libstorage/libstoragemgmt-doc/pull/16#discussion_r132961859 ?

**Comment by cathay4t** [comment]
@brraghav Never mind. I will commit another patch to fix the https://github.com/libstorage/libstoragemgmt-doc/pull/16#discussion_r132961859

**Pull request merged**